bucket/uget-portable.json

```diff
@@ -15,10 +15,14 @@
         ]
     ],
     "checkver": {
-        "url": "https://ugetdm.com/downloads/windows/",
-        "regex": "Latest Stable Version: ([\\d\\.]+)"
+        "url": "https://sourceforge.net/projects/urlget/rss?path=/uget%20%28stable%29",
+        "regex": "/uget-([\\d.]+)-win32\\+gtk3\\.7z"
     },
     "autoupdate": {
-        "url": "https://downloads.sourceforge.net/project/urlget/uget%20%28stable%29/$version/uget-$version-win32%2Bgtk3.7z"
+        "url": "https://downloads.sourceforge.net/project/urlget/uget%20%28stable%29/$version/uget-$version-win32%2Bgtk3.7z",
+        "hash": {
```

Comment:
What is wrong with normal sourceforge mode?

Reply:
I guess because the `+` in filename, `$basename` can't work. By the way, why not use `+` directly in url?

Reply:
Please try it, should work. (sf is blocked in my office...)

Reply:
Because `+` is reserved entity. It is used to concat string.

`https://....?scriptparams_=-k+PRTGAIH>` will result in `scriptsparams_` property contains string `-k PRTGAIH`

Reply:
@Ash258 Thanks. 
@niheaven It does not work, as you can see, the hash is sha256 before.